app/controllers/courses/gcse_requirements_controller.rb

```diff
@@ -61,17 +69,13 @@ def translate_params(key)
       end
     end
 
-    def build_course
-      @course = Course
-        .includes(:provider)
-        .where(recruitment_cycle_year: params[:recruitment_cycle_year])
-        .where(provider_code: params[:provider_code])
-        .find(params[:code])
-        .first
-    end
-
     def redirect_to_basic_details_page_if_provider_is_not_in_the_2022_cycle_or_higher
       redirect_to provider_recruitment_cycle_course_path unless @course.provider.recruitment_cycle_year.to_i >= Provider::CHANGES_INTRODUCED_IN_2022_CYCLE
     end
+
+    def copy_field_if_present_in_source_course(field)
```

Comment:
This has already been defined here https://github.com/DFE-Digital/publish-teacher-training/pull/1895/files#diff-893958d0871030808a878270eb8f9d525dee396118c436ba7443f822f455cec0R16. Are they the same?